spec/services/people/importer_spec.rb

```diff
@@ -0,0 +1,111 @@
+RSpec.describe People::Importer do
+  subject(:service) { described_class.new(nomis_attributes) }
+
+  let(:nomis_attributes) do
+    {
```

Comment:
This is taken from a real response in staging so is as ecologically valid as possible.